extensions/script-libraries/mime/mime.livecodescript

```diff
@@ -0,0 +1,1271 @@
+﻿script "com.livecode.script-library.mime"
+/*
+Copyright (C) 2016 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>. 
+*/
+
+on revLoadLibrary
+   if the target is not me then
+      pass revLoadLibrary
+   end if
+   
+   insert the script of me into back
+   
+   if the environment contains "development" then
+      set the _ideoverride of me to true
+      revSBAddAvailableLibrary the short name of me, "MIME"
+   end if
+   
+   __Initialize
+end revLoadLibrary
+
+on revUnloadLibrary
+   if the target is not me then
+      pass revUnloadLibrary
+   end if
+   
+   remove the script of me from back
+end revUnloadLibrary
+
+/**
+
+This script library implements portions of Multipurpose Internet Mail Extensions (MIME)
+as described in RFCs 2045 - 2049
+
+*/
+
+local sMIMETypes
+
+/**
+
+The MIME version this implemented in the library
+
+Example:
+put "MIME-Version:" && MIMEVersion() & crlf after tHeaders
+
+Description:
+Use the <MIMEVersion> function create a `MIME-Version` header.
+
+*/
+
+function MIMEVersion
+   return 1.0
```

Comment:
OK and Hmm... I generally avoid `Get` for functions as it's both redundant and less english like to write `put GetSomething() into container` rather than `put Something() into container`. It does make more sense in other languages where its `something = GetSomething()`. But assuming it's policy then do you think I should stick with the standardised `MIME` prefix for the library making this `MIMEGetVersion()`